package.json

```diff
@@ -34,7 +34,7 @@
     "clipboard": "^2.0.4",
     "connect": "^3.6.6",
     "eslint": "^5.16.0",
-    "govuk-frontend": "^3.11.0",
+    "govuk-frontend": "github:alphagov/govuk-frontend#1c97e74e",
```

Comment:
I don't think we want this change, do you mind if I edit the commit to remove it?

Reply:
I think this is in here (along with `$govuk-new-link-styles`) because the branch is 121 commits behind main, so doesn't include the latest changes including the new link styles. The branch will need to be rebased anyway as there are conflicts.

Reply:
Yes @36degrees that's true

Reply:
I can do that :)